products/opensuse/main.pm

```diff
@@ -278,6 +278,7 @@ sub load_inst_tests {
         if (get_var("DOCRUN") || get_var("IMPORT_USER_DATA") || get_var("ROOTONLY")) {    # root user
             loadtest "installation/user_settings_root";
         }
+        loadtest "installation/disable_grub_timeout";
```

Comment:
can you move that after "installation_overview_before"? Then it would be equivalent to "sle"